### DISCUSSION TITLE
[Mobile app] Mobile app with automatic activity upload to Strava

### DISCUSSION THREAD

**Discussion by EnergyStaRed** [CTHRU/Huawei-TCX-Converter#46] [Ideas]
It's not very practical to export activities from Huawei Health then send to PC, convert using Hitrava and then upload to Strava.
Much better would be automatic export, transform and upload via mobile app. As soon as you complete an activity it should automatically be exported to Hitrava and further uploaded to Strava.

**Comment by JonnIEnglish** [comment]
Agreed, just not sure how the best way to go about that would be...

**Reply by megamalex** [reply]
Is it possible to use the Huwei Health Kit to achieve the same outcome calling an API ? I just looked at registering noting I needed 2 proofs of identity passport/drivers license and bank card (?) "A bank document refers to a bank card, bankbook, or online/offline bank statement with your full name. A landscape image is preferred.
This will be used only to verify your identity and then deleted immediately."
